apm-lambda-extension/extension/process_env.go

```diff
@@ -101,7 +103,7 @@ func ptrFromString(v string) *string {
 }
 
 // ProcessEnv extracts ENV variables into globals
-func ProcessEnv(manager secretManager) *extensionConfig {
+func ProcessEnv(manager secretManager, Log *zap.SugaredLogger) *extensionConfig {
```

Comment:
```suggestion
func ProcessEnv(manager secretManager, log *zap.SugaredLogger) *extensionConfig {
```

(and rename references to Log below)

Reply:
Good catch!